src/fidesops/schemas/storage/storage.py

```diff
@@ -213,13 +215,13 @@ def json_validator(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         return values
 
 
-class StorageDestinationResponse(BaseModel):
+class StorageDestinationResponse(FidesKeyMixin):
     """Storage Destination Response Schema"""
 
     name: str
     type: StorageType
     details: Dict[StorageDetails, Any]
-    key: str
+    key: Optional[FidesKey]
```

Comment:
It's not optional in the response, right? We've created the key for them if it doesn't exist and are returning it.

Reply:
right, I'll fix this!